examples/vue-wagmi/package.json

```diff
@@ -7,8 +7,8 @@
     "build:examples": "vite build"
   },
   "dependencies": {
-    "@wagmi/connectors": "4.3.9",
-    "@wagmi/core": "2.9.7",
+    "@wagmi/connectors": "5.0.3",
```

Comment:
Seems like a major version upgrade. Do we sure this won't break anything or did we make related changes? Do we have a migration source for it?

Reply:
sorted most breaking changes 
https://www.smartwallet.dev/sdk/v3-to-v4-changes